rascal-lsp/src/main/java/org/rascalmpl/vscode/lsp/util/locations/Locations.java

```diff
@@ -35,7 +35,7 @@
 import org.eclipse.lsp4j.TextDocumentItem;
 import org.rascalmpl.uri.URIResolverRegistry;
 import org.rascalmpl.uri.URIUtil;
-
+import org.rascalmpl.values.parsetrees.TreeAdapter;
```

Comment:
unused?

Reply:
yup, this code was moved, I'll cleanup :)